config/samples/simplenificluster.yaml

```diff
@@ -38,49 +83,23 @@ spec:
   propagateLabels: true
   nifiClusterTaskSpec:
     retryDurationMinutes: 10
+  readOnlyConfig:
+    zookeeperProperties:
+      overrideSecretConfig:
+        data: zookeeper.properties
+        name: simplenifi
```

Comment:
If the intent is for this to work out of the box on a clean cluster, then i believe this `overrideSecretConfig` will cause it to fail. the `simplenifi` secret would need to exist in the nifi namespace for this to work.

Reply:
I do not fail, but I removed it as the cluster can run without this.